_site/assets/js/just-the-docs.js

```diff
@@ -59,7 +59,7 @@ jtd.getTheme = function() {
 
 jtd.setTheme = function(theme) {
   var cssFile = document.querySelector('[rel="stylesheet"]');
-  cssFile.setAttribute('href', '/js-visualize-components-samples/assets/css/just-the-docs-' + theme + '.css');
+  cssFile.setAttribute('href', 'http://localhost:4000/js-visualize-components-samples/assets/css/just-the-docs-' + theme + '.css');
```

Comment:
should we push only docs after executing `bundle exec jekyll build` locally?
to avoid pushing `localhost` to the repo?